Support JPMS?

Can we have your support for the JPMS?

* Add `module-info.java`
* Or put an [`Automatic-Module-Name`](http://branchandbound.net/blog/java/2017/12/automatic-module-name/)

Thank you.

@laurentschoelens No, sir. The module info is also required in build time.

```
[ERROR]   (package org.jvnet.jaxb.lang is declared in the unnamed module, but module com.github.jinahya.spdx.license.list does not read it)
```

--- a/jaxb-plugins-parent/jaxb-plugins-runtime/pom.xml
+++ b/jaxb-plugins-parent/jaxb-plugins-runtime/pom.xml
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>org.eclipse.angus</groupId>
       <artifactId>angus-activation</artifactId>
-      <scope>provided</scope> 
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
@@ -54,6 +54,9 @@
         <configuration>
           <archive>
             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
+              <Automatic-Module-Name>org.jvnet.jaxb.plugins.runtime</Automatic-Module-Name>
+            </manifestEntries>
           </archive>
         </configuration>
       </plugin>

